### PR TITLE
Slack

### PR DESCRIPTION
**Pull request by lsk567**
This PR contains the following changes:
1. Add the lf-community slack channel to the community page;
2. Add slack to the footer;
3. Add Shaokai as a contributor;
4. Maintain an alphabetical order of the contributors.

The resulting page looks like this:
slack-community

### CODE REVIEW

**Comment by lhstrh** [comment]
Looks great! Thanks!

File: packages/lingua-franca/src/components/layout/SiteFooter.tsx [hunk 1]
```diff
@@ -48,6 +48,10 @@ const communityLinks = [
     title: "Web Repo",
     url: "https://github.com/lf-lang/website-lingua-franca",
   },
+  {
+    title: "Slack",
+    url: "https://join.slack.com/t/lf-community/shared_invite/zt-1b4egenxd-Dz~uG5Ps6ce71pgJG02cjw",
```
Comment (lhstrh) [inline packages/lingua-franca/src/components/layout/SiteFooter.tsx:53]: Don't these kinds of invite URLs have a limited life span?

File: packages/lingua-franca/src/templates/pages/css/community.scss [hunk 2]
```diff
@@ -85,6 +85,10 @@
         background-color: lighten(black, 30%);
       }
       
+      &.slack {
+        background-image: url("../../../assets/community/connect-slack-icon.svg");
+        background-color: lighten(rgb(167, 167, 167), 30%);
```
Comment (lhstrh) [inline packages/lingua-franca/src/templates/pages/css/community.scss:90]: Would this work as expected in the Dark theme as well?

**Review by lhstrh — APPROVED** [review]

File: packages/lingua-franca/src/components/layout/SiteFooter.tsx (hunk 1 above)
Comment (lsk567) [reply packages/lingua-franca/src/components/layout/SiteFooter.tsx:53]: Yes, but this link, specifically, is set to never expire.

File: packages/lingua-franca/src/templates/pages/css/community.scss (hunk 2 above)
Comment (lsk567) [reply packages/lingua-franca/src/templates/pages/css/community.scss:90]: To me, this looks just as good in the Dark theme. Picture below
slack-dark

**Pull request merged**